source/backend/api/Services/DispositionFileService.cs

```diff
@@ -431,6 +431,64 @@ public List<DispositionFileExportModel> GetDispositionFileExport(DispositionFilt
                 }).ToList();
         }
 
+        public PimsDispositionFile UpdateProperties(PimsDispositionFile dispositionFile, IEnumerable<UserOverrideCode> userOverrides)
+        {
+            _logger.LogInformation("Updating disposition file properties...");
+            _user.ThrowIfNotAuthorized(Permissions.DispositionEdit, Permissions.PropertyView, Permissions.PropertyAdd);
```

Comment:
this is quite subtle but this extension method will NOT enforce all of those claims. If the user has **any** of those it will let if pass. The one you want is called something like ThrowIfNotAll or something. 
Alternatively, you could list individual claim checks like in 3 lines of ThrowIfNotAuthorized

I think the naming is confusing and we should address with some tech-debt (not this pr of course)